basics/runtime/src/main/java/org/jvnet/jaxb2_commons/lang/DefaultCopyStrategy.java

```diff
@@ -267,6 +270,74 @@ protected Object copyInternal(ObjectLocator locator, Cloneable object) {
 			}
 		}
 	}
+	protected Object copyInternalJdk11(ObjectLocator locator, Cloneable object) {

```

Comment:
This method has a lot of copy+paste from the other copyInternal.  Perhaps we can just add an argument to adjust behavior to the original method?

Reply:
I didn't want to change existing behaviour for jdk8 / jdk11 which makes it works well.
The exception is only thrown with jdk17 since jdk16 has locked more aggressively the internal jdk classes.
This can be the default impl with jakarta-jaxb4 since baseline will be jdk11 (jaxb4 requires jdk11)